README.md

```diff
@@ -12,70 +12,193 @@ Airport Challenge
+| Plane   |planeID @String                |                   |         |
+
+**Tests**  
+*Test 1* - Check that only Plane types are able to land at the Airport when planeLands  
+*Test 2* - Length of planesAtAirport array at the Airport increases by 1 when planeLands  
+*Test 3* - Check that edge cases such as 'null' as a type are unable to land at the airport and length of planesAtAirport remains at 0  
+*Test 4* - Check that when something that isn't a plane attempts to land at the airport, a message is returned  
+*Test 5* - When a new airport instance is created from the Airport class, it contains an airportName property  
+*Test 6* - When a new plane instance is created from the plane class, it contains a planeID property  
+
+## User Story 2  
+As the system designer  
+So that the software can be used for many different airports  
+I would like a default airport capacity that can be overridden as appropriate  
+
+**Domain Model 2**
+
+| Objects | Properties                   | Messages          | Outputs |
+| ------- | -------------------------    | ----------------- | ------- |
+| Airport |planesAtAirport @Array[@Plane]| planeLands(@Plane)| @Void   |
+|         |airportName@String            |                   |         |
+|         |airportCapacity @Integer      |                   | @String |
+| Plane   |planeID @String               |                   |         |
+
+**Tests**  
+
+*Test 1* - Check the value returned for default airport capacity  
+*Test 2* - Check the value returned when capacity changed with positive integer  
+*Test 3* - Return an error message when capacity is inputted as below 0  
+*Test 4* - Return an error message when capacity is inputted as null  
+*Test 5* - Return an error message when capacity is inputted as a non-numerical string  
+  
+To note: For tests 2.3 - 2.5, I wanted to throw an error rather than set the capacity as a string, however I then couldn't run the tests.  
+
+## User Story 3  
+As an air traffic controller  
+To ensure safety  
+I want to prevent landing when the airport is full  
+
+**Domain Model 3**
+
+| Objects | Properties                   | Messages          | Outputs |
+| ------- | -------------------------    | ----------------- | ------- |
+| Airport |planesAtAirport @Array[@Plane]| planeLands(@Plane)| @Void   |
+|         |airportName@String            |                   |         |
+|         |airportCapacity @Integer      | atCapacity()      |@Boolean |
+| Plane   |plane id @String              |                   |         |
+
+**Tests**  
+
+*Test 1* - When the length of planesAtAirport and airportCapacity are the same, the test returns true  
+*Test 2* - When the length of planesAtAirport and airportCapacity are the same (1), atCapacity returns true  
+*Test 3* - When the length of planesAtAirport and airportCapacity are different, atCapacity returns false  
+*Test 4* - When atCapacity is true, planeLands returns an error message  
```

Comment:
I should have added a test for the atCapacity function if an incorrect value had been inputted for airportcapacity and so was set as a string error message